services/workflows-service/src/case-management/controllers/case-management.controller.ts

```diff
@@ -138,12 +162,20 @@ export class CaseManagementController {
         const isMonitored = checkIsMonitored();
         const matches = getMatches();
 
+        const businesses = endUser.businesses?.length
+          ? endUser.businesses.map(business => business.companyName).join(', ')
+          : endUser.endUsersOnBusinesses
+              ?.map(endUserOnBusiness => endUserOnBusiness.business.companyName)
+              .join(', ');
+
         return {
           correlationId: endUser.correlationId,
           createdAt: endUser.createdAt,
           name: `${endUser.firstName} ${endUser.lastName}`,
-          businesses: endUser.businesses?.map(business => business.companyName).join(', '),
-          role: 'UBO',
+          businesses,
+          roles: endUser.endUsersOnBusinesses?.flatMap(
+            endUserOnBusiness => endUserOnBusiness.position,
```

Comment:
Can `position` be `undefined` or an empty string?

Reply:
I didn't want to force the use of position as in some cases we might not have it